packages/Algernon.package/ALGItemFilter.class/instance/accepts..st

```diff
@@ -2,5 +2,5 @@ filtering
 accepts: aStringCollection
 
 	(aStringCollection size <= 1)
-		ifTrue: [ ^ false ].
-	^ self class keywords includes: aStringCollection first
\ No newline at end of file
+		ifFalse: [ ^ self class keywords includes: aStringCollection first ].
+	^ false
```

Comment:
This should be `(aString collection size > 1) and: [ self class keywords includes: aStringCollection first ]`